frontend/src/metabase/dashboard/actions/tabs.ts

```diff
@@ -151,7 +161,7 @@ export const tabsReducer = createReducer<DashboardState>(
       (state, { payload: { tabId } }) => {
         const { dashId, prevDash, prevTabs } = getPrevDashAndTabs({ state });
         if (!dashId || !prevDash) {
-          throw Error(
+          throw new Error(
```

Comment:
refactored this on all the file after a suggestion to keep it consistent with how we do in the rest the codebase